server/chat-plugins/info.js

```diff
@@ -2380,6 +2387,70 @@ const commands = {
 	],
 };
 
+/** @type {PageTable} */
+const pages = {
+	punishments(query, user, connection) {
+		this.title = 'Punishments';
+		let buf = "";
+		this.extractRoom();
+		if (!user.named) return Rooms.RETRY_AFTER_LOGIN;
+		buf += `<div class="pad"><h2>List of active punishments:</h2>`;
+		if (!user.can('mute')) {
+			return buf + `<div class="notice message-error">Access denied.</div>`;
+		}
+		if (!this.room.chatRoomData) {
+			return buf + `<div class="notice message-error">This page is unavailable in temporary rooms / non-existent rooms.</div>`;
+		}
+		const store = new Map();
+		const muteQueue = this.room.muteQueue;
```

Comment:
Why? You're only using this var twice it's not a big deal to do this.room.muteQueue for those